### DISCUSSION TITLE
Error [ERR_REQUIRE_ESM]: require() of ES Module /node_modules/title-case/dist/index.js from /build/index.js not supported.

### DISCUSSION THREAD

**Discussion by 0x5am5** [webstudio-is/webstudio-community#62] [Q&A]
I get the below error when trying to run my project with the webstudio cli

```
Error [ERR_REQUIRE_ESM]: require() of ES Module /node_modules/title-case/dist/index.js from /build/index.js not supported.
```

I have tried using nodejs version 18.0.0 and latest 20.8.0 to no avail.

**Comment by 0x5am5** [comment]
Soved by add `title-case` and `change-case` to remix.config.js.

```js
/** @type {import('@remix-run/dev').AppConfig} */
module.exports = {
  ignoredRouteFiles: ["**/.*"],
  serverModuleFormat: "cjs",
  serverDependenciesToBundle: [
    /@webstudio-is\//,
    "nanoid",
    "@jsep-plugin/assignment",
    "title-case",
    "change-case",
  ],
  future: {
    v2_errorBoundary: true,
    v2_headers: true,
    v2_meta: true,
    v2_normalizeFormMethod: true,
    v2_routeConvention: true,
    v2_dev: true,
  },
};
```

**Reply by TrySound** [reply]
Good catch! Would you like to send a PR with fix?

**Reply by 0x5am5** [reply]
Is this _actually_ the answer though or is something bigger the issue? I don't feel confident in that.

**Reply by TrySound** [reply]
The fix in your repo is correct though will be overwritten with another `$ webstudio build`.
You can do same change for builtin templates.
https://github.com/webstudio-is/webstudio/tree/main/packages/cli/templates

**Reply by 0x5am5** [reply]
Yes precisely. I’m not suggesting this is a long term fix and I don’t know the repo well enough to feel comfortable putting in a PR.

I’m not sure what you’re asking of me lol!

**Reply by TrySound** [reply]
Only suggested to contribute. No worries we will fix ourselves.

**Reply by 0x5am5** [reply]
Thanks, I appreciate that. I would feel more confident if i knew the codebase.

**Comment by TrySound** [comment]
Hey, the fix has been released.
